### DISCUSSION TITLE
AI Assist to build with Falcon

### DISCUSSION THREAD

**Discussion by wadhia-yash** [falconry/falcon#2307] [Show and tell]
Hi there, 

Falcon is a popular framework for building mission-critical REST APIs and microservices. But most copilots aren't able to help with the latest syntax. So we created a custom AI Assist trained on this repo code + issues.

Please try it [here](https://app.commanddash.io/agent/github_falconry_falcon)

It is also mentionable as a badge in the README and can help developers landing on the repo build apps for their use cases.

<a href="https://app.commanddash.io/agent/github_falconry_falcon"><img src="https://img.shields.io/badge/AI-Code%20Agent-EB9FDA"></a>

```
<a href="https://app.commanddash.io/agent/github_falconry_falcon"><img src="https://img.shields.io/badge/AI-Code%20Agent-EB9FDA"></a>
```

hope this helps!

**Comment by vytas7** [comment]
Hi @wadhia-yash!
We have just tried this briefly, and I must say I am really impressed for the most of it! Generic GPTs are indeed at times a bit out of touch wrt the newest developments in Falcon.

@CaselIT encountered some issue with code not rendering correctly, but it looks OK for me in Firefox.

That said, I am a bit leery of adding a badge to our README at this time. We probably need to understand your business model and the product better in order to make this decision; also, we normally only add top-level badges, logotypes, sponsored links to our README, website, and docs in conjunction with the [Silver Membership](https://opencollective.com/falcon)+.

We will try to add this to our docs for free, if that works for you as a starting point?

**Reply by wadhia-yash** [reply]
Hi @vytas7 

We're glad to hear you liked the results! The badge is our free service to the open-source community and we don't monetize it in any form :slightly_smiling_face: The model results are also completely secure and we don't retain users' code.

Please feel free to add the badge where you believe the users can quickly access and best benefit from it. 

We're available to provide any support!
